aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -1934,7 +1945,7 @@ Tensor median_mps(const Tensor& input_t) {
     auto stream = at::mps::getCurrentMPSStream();
 
     @autoreleasepool {
-        string key = func_name + ":" + to_string(dim_) + ":" + native_mps::getTensorsStringKey(input_t);
+        string key = func_name + ":" + to_string(dim_) + ":" + native_mps::getTensorsStringKey(input_t) + ":" + native_mps::getTensorsStringKey(indices_t);
```

Comment:
Please change to this (the getTensorsStringKey() accepts a TensorList so you can pass multiple ones to it):
```suggestion
        string key = func_name + ":" + to_string(dim_) + ":" + native_mps::getTensorsStringKey({input_t, indices_t});
```

Reply:
Sure!